src/riotwatcher/_apis/team_fight_tactics/urls/DataDragonUrls.py

```diff
@@ -0,0 +1,14 @@
+from ...league_of_legends.urls.DataDragonUrls import DataDragonEndpoint, DDragonVersionLocaleEndpoint
```

Comment:
dont import things from league APIs in TFT; copy+pasting classes is preferable here so that the games are not mixed up in any way